package.json

```diff
@@ -2,17 +2,18 @@
   "name": "open-ai-reviewer",
   "version": "1.0.0",
   "description": "Open AI powered code reviews",
-  "main": "lib/main.js",
+  "main": "lib/index.js",
   "author": "Ville Saukkonen",
   "license": "MIT",
   "scripts": {
-    "build": "tsc",
-    "package": "ncc build --source-map --license licenses.txt",
+    "build": "tsc --sourceMap",
+    "package": "ncc build src/index.ts --source-map --license licenses.txt",
```

Comment:
The package script has been modified to specify 'src/index.ts' as the entry point. Ensure that this file exists and is the intended entry point for the build process. Additionally, verify that the use of '--source-map' is compatible with the output format.